css-colors/styles.css

```diff
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: rgb(244 249 251);
+}
+
+section {
+  padding: 1.5rem 0;
+}
```

Comment:
Please adjust the background color of the `section`, not the `body`.

---

![image](https://user-images.githubusercontent.com/16143771/176497867-6e04f60d-cc15-4999-9b2c-de74dc703729.png)